crawl-ref/source/mon-info.cc

```diff
@@ -1825,6 +1783,502 @@ void get_monster_info(vector<monster_info>& mons)
+            }
+	});
+    toret.push_back({"insane", "insane", 
+	    [](const monster_info &mi, bool newconditions)
+            { 
+                return mi.is(MB_INSANE);
+            }
+	});
+    toret.push_back({"stupefied", "stupefied",
+	    [](const monster_info &mi, bool newconditions)
+            {
+                return mi.is(MB_DUMB);
+            }
+	});
+    toret.push_back({"paralysed", "paralysed",
+	    [](const monster_info &mi, bool newconditions)
+            {
+                return mi.is(MB_PARALYSED);
+            }
+	});
+    toret.push_back({"caught", "caught",
+	    [](const monster_info &mi, bool newconditions)
+            {
+                return mi.is(MB_CAUGHT);
+            }
+	});
+    toret.push_back({"webbed", "webbed",
+	    [](const monster_info &mi, bool newconditions)
+            {
+                return mi.is(MB_WEBBED); 
+            }
+	});
+    toret.push_back({"petrified", "petrified",
+	    [](const monster_info &mi, bool newconditions)
+            {
+                return mi.is(MB_PETRIFIED); 
+            }
+	});
+    toret.push_back({"pinned", "pinned", 
+	    [](const monster_info &mi, bool newconditions)
+            {
+                return mi.is(MB_PINNED); 
+            }
+	});
+    toret.push_back({"polearm","polearms",
+	    [](const monster_info &mi, bool newconditions)
+            {
+                if (mi.itemuse() >= MONUSE_STARTING_EQUIPMENT)
+                {
+                    const item_def* weapon = mi.inv[MSLOT_WEAPON].get();
+                    if (weapon && weapon_reach(*weapon) == REACH_TWO)
+                        return newconditions;
+                }
+                if (mi.type == MONS_DANCING_WEAPON && 
+                    mi.reach_range() == REACH_TWO) 
+                    return newconditions;
+                return false;
+            }
+	});
+    toret.push_back({"launcher","launcher", 
+	    [](const monster_info &mi, bool newconditions)
+            {
+                if (mi.itemuse() >= MONUSE_STARTING_EQUIPMENT)
+                {
+                    const item_def* weapon = mi.inv[MSLOT_WEAPON].get();
+                    if (weapon && is_ranged_weapon_type(weapon->sub_type))
+                        return newconditions;
+                }
+                return false;
+            }
+	});
+    // We know our own friendly summons are summons, right?
+    toret.push_back({"summoned", "summoned",
+	    [](const monster_info &mi, bool newconditions)
+            { 
+                return newconditions && mi.is(MB_SUMMONED) &&
+                (mi.attitude != ATT_FRIENDLY); 
+            }
+	});
+    toret.push_back({"scary", "scary", 
+	    [](const monster_info &mi, bool newconditions)
+            { 
+                return newconditions && mi.is(MB_FEAR_INSPIRING); 
+            }
+	});
+    // Bah, these next four are really too long
+    toret.push_back({"chanting recall", "chanting recall",
```

Comment:
"recalling" perhaps?

Reply:
I considered and rejected "recalling" since it sounds like the monster's trying to remember where it put its keys. :-/